worldwind/src/commonMain/kotlin/earth/worldwind/shape/Placemark.kt

```diff
@@ -461,13 +459,7 @@ open class Placemark @JvmOverloads constructor(
         vertexArray[4] = (placePoint.y - groundPoint.y).toFloat()
         vertexArray[5] = (placePoint.z - groundPoint.z).toFloat()
 
-        // Regenerate vertex buffer on array change
-        val hashCode = vertexArray.contentHashCode()
-        if (vertexArrayHashCode != hashCode) {
-            vertexArrayHashCode = hashCode
-            vertexBufferKey = nextCacheKey()
-        }
-        drawable.vertexPoints = rc.getBufferObject(vertexBufferKey) { FloatBufferObject(GL_ARRAY_BUFFER, vertexArray) }
+        drawable.vertexPoints = vertexArray
```

Comment:
We do not need vertexArray in this class. Syut preefine DrawableLines.vertexPoints as FloatArray(6) and ssign its elements directly https://github.com/NASAWorldWind/WorldWindAndroid/blob/0dacfec3da57b42a5228d99ca862a6ec201e2800/worldwind/src/main/java/gov/nasa/worldwind/shape/Placemark.java#L896

If you keep current approach we will have trouble with multiple globes, as you copy array refernece into dravable, but only valuse should be copied. So just remove vertexArray and fill predefined vertexPonts array in drawale.

Reply:
++